include/tvm/relay/pass.h

```diff
@@ -2,18 +2,202 @@
  *  Copyright (c) 2018 by Contributors
  * \file tvm/relay/pass.h
  * \brief The set of Relay passes written in C++.
+ *
+ * This file also implements a pass manager. The pass manager manages a sequence
+ * of Relay-to-Relay transformation passes over a particlar unit of AST. The
+ * design is largely inspired from LLVM's pass manager.
```

Comment:
This design is inspired by LLVM's pass manager and modern deep learning frameworks that performs tensor->tensor transformaitons